subcommittees/mentoring/roles/mentees.md

```diff
@@ -0,0 +1,17 @@
+# Mentees
+
+## Responsibilities
+
```

Comment:
- Communicate with your Mentor about your goals and needs.  
- Attend scheduled meetings and participate actively or let your Mentor know you won’t be attending.  
- Be respectful of other Mentees and Mentors. Provide a safe space for questions and mistakes.